core/plugins.in

```diff
@@ -20,23 +21,61 @@ var openedFinalizePlugins map[string]Plugin
 func LoadPlugin(name string, plugintype api.PluginType, recipe *api.Recipe) (uintptr, error) {
 	fmt.Println("Loading new plugin")
 
-	localPluginPath := fmt.Sprintf("%s/%s.so", recipe.PluginPath, name)
+	projectPluginPath := fmt.Sprintf("%s/%s.so", recipe.PluginPath, name)
 
-	globalPluginPath := fmt.Sprintf("%INSTALLPREFIX%/share/vib/plugins/%s.so", name)
+	installPrefixPath := fmt.Sprintf("%INSTALLPREFIX%/share/vib/plugins/%s.so", name)
+
+	globalPluginPathsEnv, isGPPEDefined := os.LookupEnv("XDG_DATA_DIRS")
+	if !isGPPEDefined || globalPluginPathsEnv == "" {
```

Comment:
```suggestion
	globalPluginPathsEnv, isGPPEDefined := os.LookupEnv("XDG_DATA_DIRS")
	if !isGPPEDefined || len(strings.TrimSpace(globalPluginPathsEnv)) == "" {
```
just checking if its equal to `""` wouldn't work since the envvar can be just a single space or some other whitespace
also what does GPPEDefined mean? I think something like `XDDDefined` would make more sense